src/containers/CharacterProfile/index.js

```diff
@@ -60,7 +52,7 @@ class CharacterProfileScreen extends Component {
     this.state = {
       scrollHeader: false,
       searchFocus: false,
-      orientation: isPortrait() ? 'portrait' : 'landscape',
+      isPortrait: isPortrait() ? true : false,
```

Comment:
If we're already using react-native-orientation, why do we need this function?

Reply:
We aren't actually using react-native orientation, it's installed but this Util function is just using Dimensions